config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,32 @@
 # Be sure to restart your server when you modify this file.
+module ParameterFiltering
+  ATTRIBUTE_WHITELIST = %w[
```

Comment:
Shall we comment where we expect these attributes to come from? Are they params?

Reply:
Sure I'll leave a comment explaining this approach, as its a bit different from most rails apps.